Demo/Application/Features/PayPal - Native Checkout/BraintreeDemoPayPalNativeCheckoutViewController.swift

```diff
@@ -1,10 +1,28 @@
 import Foundation
 import UIKit
 import BraintreePayPalNativeCheckout
+import PayPalCheckout
```

Comment:
With this implementation, we would require merchants to directly import `PayPalCheckout` to use the shipping callback functionality. I'll defer to @jaxdesmarais and team to weigh in on this, but that doesn't feel like a great experience to me - the alternative would be to create BT types that can be translated to the types the NXO SDK expects in the shipping callback interface. What do you think?

Reply:
Agree, we should not require merchants to pull in NXO in addition to that module that contains NXO as a dependancy. I think we can either create our own BT types that can be translated to the NXO types as you mentioned or create a wrapper for the shipping callback types that can be passed through to the callback without the merchant having to pull in NXO directly.